### ISSUE TITLE
error An unexpected error occurred: "EPERM: operation not permitted, copyfile

### ISSUE THREAD

**Issue by DevDuque** [yarnpkg/yarn#8473]
Error: EPERM: operation not permitted, copyfile 'C:\Users\User\AppData\Local\Yarn\Cache\v6\npm-@bcoe-v8-coverage-0.2.3-75a2e8b51cb758a7553d6804a5932d7aace75c39-integrity\node_modules\@bcoe\v8-coverage\LICENSE.txt' -> 'D:\Montagens\Projects\k\node_modules\@bcoe\v8-coverage\LICENSE.txt'

Error creating react app

**Comment by DahirWarsame** [comment]
I recieve the same error when I run the `yarn` command

`
error An unexpected error occurred: "EPERM: operation not permitted, copyfile 'C:\\Users\\dahir\\AppData\\Local\\Yarn\\Cache\\v6\\npm-@bcoe-v8-coverage-0.2.3-75a2e8b51cb758a7553d6804a5932d7aace75c39-integrity\\node_modules\\@bcoe\\v8-coverage\\LICENSE.txt' -> 'D:\\Users\\dahir\\Documents\\Hu\\Open Innovation 2020-2021\\VCSmartLock\\mobile\\node_modules\\@bcoe\\v8-coverage\\LICENSE.txt'".
`

full error log: [https://pastebin.com/mv0AYzxC](https://pastebin.com/mv0AYzxC)

**Comment by zimmermanw84** [comment]
Interesting. I also have a similar error on a license file.

```
error An unexpected error occurred: "EPERM: operation not permitted, copyfile '/usr/local/share/.cache/yarn/v6/npm-@snyk-composer-lockfile-parser-1.4.0-a16fff515288496a27292b32b5cc14d7d84f9026-integrity/node_modules/@snyk/composer-lockfile-parser/LICENSE' -> '/var/www/soxhub/source/node_modules/@snyk/composer-lockfile-parser/LICENSE'".
```

More context...
I am currently upgrading node versions (13x -> 14.15.4LTS) so maybe it's a caching issue?

**Comment by mattbrictson** [comment]
If you are using Docker, then the solution to this EPERM error might need to upgrade the Docker version. See https://github.com/yarnpkg/yarn/issues/8389

**Comment by merceyz** [comment]
Closing as there is nothing actionable from Yarns side here

**Comment by nsavinda** [comment]
Change yarn cache path to short path  [https://classic.yarnpkg.com/en/docs/cli/cache](https://classic.yarnpkg.com/en/docs/cli/cache)
or
Copy project files to a short path like `C:\projects\projectfiles`
Then try again

**Comment by edelauna** [comment]
If you're running a container with a custom seccomp profile ensure that the following sys-call exists in the profile:

```json
{
	"name": "copy_file_range",
	"action": "SCMP_ACT_ALLOW",
	"args": []
},
```

**Comment by Ravint20** [comment]
stop your server and perform installation then start your server.

**Comment by roman-shandurenko** [comment]
> stop your server and perform installation then start your server.

@Ravint20 What do you mean by that? Could you elaborate, please?

**Comment by Sandamalt** [comment]
Terminate or stop the server ( close and reopen Vs code) and try again. this will work for me !!

**Comment by Q-org** [comment]
error An unexpected error occurred: "https://registry.yarnpkg.com/rxjs/-/rxjs-7.8.0.tgz: ESOCKETTIMEDOUT

**Comment by amineshon** [comment]
In Windows first stop the development server and then try to install package.

**Comment by ankitaeth** [comment]
C:\Code\custom-connect-flow-AA-YT-Course-main>yarn install
yarn install v1.22.19
[1/4] Resolving packages...
[2/4] Fetching packages...
error An unexpected error occurred: "EPERM: operation not permitted, unlink 'C:\\Users\\USER\\AppData\\Local\\Yarn\\Cache\\v6\\npm-@react-icons-all-files-4.1.0-477284873a0821928224b6fc84c62d2534d6650b-integrity\\node_modules\\@react-icons\\all-files\\.yarn-tarball.tgz'".
info If you think this is a bug, please open a bug report with the information provided in "C:\\Code\\custom-connect-flow-AA-YT-Course-main\\yarn-error.log".

I have also getting error  i try with administermode in cmd also then also i get error how to solve this error for this package

**Comment by RanvijayChouhan12** [comment]
**Try Removing the Node Modules Folder  Re-Install the Node Module**

this Solve my Issue

i Got this issue During Installing the Package **preline**

`yarn add preline`

**Comment by JayKCDev** [comment]
I cloned my repo from Github and ran into the same issue just now, not sure how it got fixed but still posting here for people to try it out.

1. If you have `package-lock.json` file along side `yarn.lock` delete that as it might crash out of conflict.
2. If you're running `yarn` command at two or more different directories which reside in the same parent directory then as well it might fail with the above error

In my case, it was #2, I tried running commands one after the other and it worked just fine
